README.md

```diff
@@ -1,33 +1,147 @@
+- **Streamlined Development Cycle for Production**
+    - Deploy your flow to the serving platform you choose or integrate into your app's code base easily.
+    - (Optional but highly recommended) Collaborate with your team by leveraging the cloud version of [Prompt flow in Azure AI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2).
+
+### Concept Overview
+
+[![concept](examples/tutorials/quick-start/media/concept.png)](https://microsoft.github.io/promptflow/concepts/concept-connections.html)
+
+------
+
+## Installation
+
+Ensure you have a python environment, `python=3.9` is recommended.
+
+```sh
+pip install promptflow promptflow-tools
+```
+
+## Quick Start ⚡
+
+<details>
+<summary><b>Create a chatbot with prompt flow</b></summary>
+ It creates a new <b>flow folder</b> named my_chatbot and generates the necessary flow files within it. The --flow argument is the path to the flow folder.
+</details>
+
+Initiate a prompt flow from a chat template:
+
+```sh
+pf flow init --flow ./my_chatbot --type chat
+```
+
+<details>
+<summary><b>Setup a connection for your API key</b></summary>
+Navigate to the `my_chatbot` folder, you can find a yaml file named `openai.yaml` file, which is the definition of the connection to store your Open AI key.
+</details>
+
+> <details>
+> <summary>For Azure OpenAI key, establish the connection by running:</summary>
+>
+> ```sh
+> # Override keys and endpoint with --set to avoid yaml file changes
+> pf connection create --file ./my_chatbot/azure_openai.yaml --set api_key=<your_api_key> api_base=<your_api_endpoint>
+> ```
+> </details>
+
+For OpenAI key, establish the connection by running:
+
+
+```sh
+# Override keys with --set to avoid yaml file changes
+pf connection create --file ./my_chatbot/openai.yaml --set api_key=<your_api_key>
+```
+
+<details>
+<summary><b>Chat with your flow</b></summary>
+In the `my_chatbot` folder, there's a `flow.dag.yaml` file that outlines the flow, including inputs/outputs, tools, nodes, etc. Note we're using the connection named `open_ai_connection` in the `chat` node.
+</details>
+
+> <details>
+> <summary>For Azure Open AI users, modify this file accordingly.</summary>
+> Replace the 'node:' section with following content and specify the 'deployment_name' to the model deployment you'd like to use.
+>
+> ```yaml
+> nodes:
+> - name: chat
+>   type: llm
+>   source:
+>     type: code
+>     path: chat.jinja2
+>   inputs:
+>     deployment_name: <your_azure_open_ai_deployment_name>
+>     max_tokens: '256'
+>     temperature: '0.7'
+>     chat_history: ${inputs.chat_history}
+>     question: ${inputs.question}
+>   api: chat
+>   connection: azure_open_ai_connection
+> ```
+>  </details>
+
+For OpenAI users, interact with your chatbot by running: (press `Ctrl + C` to end the session)
+
+```sh
+pf flow test --flow ./my_chatbot --interactive
+```
+
+Then you will see the chatbot in action:
```

Comment:
No need this gif, the sample is not proper.

Reply:
I feel Clement‘s opinion is very good. GIF can predict what users will see, which has a certain appeal. Here for this gif, I want to highlight our interactive run and chat history.

What sample you think is proper? I can refine it.

Reply:
we can make it in another PR until we find proper sample

Reply:
sure